packages/ckeditor5-package-tools/lib/utils/webpack-utils.js

```diff
@@ -64,5 +64,16 @@ module.exports = {
 				]
 			};
 		}
+	},
+	getModuleResolutionPaths: packageRootDir => {
+		return [
+			'node_modules',
+			//  Root of this package.
+			path.resolve( packageRootDir, 'node_modules' ),
+			// Root of the CPG repository.
+			path.resolve( packageRootDir, '..', '..', 'node_modules' ),
+			// Root of the repository that uses this package.
+			path.resolve( packageRootDir, '..', '..', '..', 'node_modules' )
```

Comment:
```suggestion
			'node_modules',
			//  Root of this package.
			path.resolve( packageRootDir, 'node_modules' ),
			// Root of the CPG repository.
			path.resolve( packageRootDir, '..', '..', 'node_modules' ),
			// Root of the repository that uses this package.
			path.resolve( packageRootDir, '..', '..', '..', 'node_modules' )
```

I am pretty sure that the CPG repository does not exist when a user created a custom package using the tool.

We should check two directories:

* a nested `node_modules/` in pkg-tools,
* If not found, check the root `node_modules/` directory.

```
.
├── node_modules/ # --- (2) then, look here\
│   ├── ...
│   ├── @ckeditor/
│   │   ├── ...
│   │   └── ckeditor5-package-tools/ # --- (1) look here.
│   │       ├── node_modules/
│   │       └── package.json
│   └── ...
└── package.json
```

To verify if everything works as we expect, I suggest creating a ZIP archive containing the `@ckeditor/ckeditor5-package-tools` package. Then, update a generated `package.json` file in a new package to force installing pkg-tools from the ZIP archive. We aim to use our local changes and simulate installing packages from npm.

When creating a new package, do not include the `--dev` option.

Reply:
I've left a minimal list of paths that passes CI, but I'm having problems with my local environment, and I'm unsure whether this will work correctly. Could someone else doublecheck?

Reply:
Let's do it tomorrow.

I think the proposed change could help.

```
getModuleResolutionPaths: packageRootDir => {
	return [
		// `{root}/node_modules/@ckeditor/ckeditor5-package-tools/node_modules`
		path.join( __dirname, '..', '..' 'node_modules' ),
		// `{root}/node_modules`
		path.resolve( packageRootDir, 'node_modules' )
	];
}
```